xds/third_party/googleapis/import.sh

```diff
@@ -24,6 +24,7 @@ SOURCE_PROTO_BASE_DIR=googleapis
 TARGET_PROTO_BASE_DIR=src/main/proto
 FILES=(
 google/api/expr/v1alpha1/syntax.proto
+google/api/expr/v1alpha1/checked.proto
```

Comment:
You can also remove the import changes in this PR.

Reply:
Sure!

Reply:
However, I think this branch will not compile after I removed this dependency. Should I wait till the other PR got merged so this will no longer be a diff?